router/router.go

```diff
@@ -217,12 +218,14 @@ func New(cfg *config.Configuration) (r *Router, err error) {
 	exchanges = newExchangeMap(cfg)
 	theExchange := exchange.NewExchange(theClient, pbc.NewClient(&cfg.CacheURL), cfg, r.MetricsEngine, bidderInfos, gdprPerms)
 
-	openrtbEndpoint, err := openrtb2.NewEndpoint(theExchange, paramsValidator, fetcher, cfg, r.MetricsEngine, pbsAnalytics, disabledBidders, defReqJSON)
+	rateConverter := currencies.NewRateConverter(&http.Client{}, cfg.RatesConverter.FetchURL, time.Duration(cfg.RatesConverter.FetchIntervalSeconds)*time.Second)
```

Comment:
Any reason this doesn't use `theClient`?

From [the docs](https://golang.org/pkg/net/http/):

> Clients and Transports are safe for concurrent use by multiple goroutines and for efficiency should only be created once and re-used.

Also... if you don't set the `Transport.TLSClientConfig` the same way `theClient` does, I believe PBS hosts who use an `https://` URL inside the Docker container might get errors.

Reply:
Oh, no, no reasons, I just replaced it to use `theClient `